Makefile

```diff
@@ -25,6 +25,7 @@ ifeq ($(GOOS),windows)
 	IS_EXE := .exe
 endif
 MINISHIFT_BINARY ?= $(GOPATH)/bin/minishift$(IS_EXE)
+TEST_TAGS ?= basic
```

Comment:
So here is an idea, why don't we make this variable "CUCUMBER_OPTS" and make it per default the empty string. Then we pass the whole _CUCUMBER_OPTS_ to the _go test_ command. This means per default not tags are applied.

If you then want to run with tags, you do:

    $ make integration CUCUMBER_OPTS="--tags basic,smoke,whatever"

This might be also more generic in case we want to pass parameters.